package.json

```diff
@@ -7,7 +7,7 @@
   "license": "N/A",
   "private": true,
   "scripts": {
-    "dev": "cross-env TESTING_MODE=true APPINSIGHTS_INSTRUMENTATIONKEY=2f1d6a9c-1b88-4f3d-bba1-64b11ffd2362 GET_SUBMISSIONS_URL=https://dddperth-functions-test.azurewebsites.net/api/GetSubmissions SUBMIT_VOTE_URL=https://dddperth-functions-test.azurewebsites.net/api/SubmitVote GET_AGENDA_URL=https://dddperth-functions-test.azurewebsites.net/api/GetAgenda SUBMIT_FEEDBACK_URL=https://dddperth-functions-test.azurewebsites.net/api/SubmitFeedback NODE_OPTIONS=--inspect next",
+    "dev": "cross-env TESTING_MODE=true APPINSIGHTS_INSTRUMENTATIONKEY=2f1d6a9c-1b88-4f3d-bba1-64b11ffd2362 GET_SUBMISSIONS_URL=https://dddperth-functions-test.azurewebsites.net/api/GetSubmissions SUBMIT_VOTE_URL=https://dddperth-functions-test.azurewebsites.net/api/SubmitVote GET_AGENDA_URL=https://dddperth-functions-test.azurewebsites.net/api/GetAgenda SUBMIT_FEEDBACK_URL=https://dddperth-functions-test.azurewebsites.net/api/SubmitFeedback next",
```

Comment:
Next is not happy about have the inspect passed this way. Cross env doesn't like it being passed how next wants it. So we have a stand off. Will look at it again later.

Reply:
Fair enough. That's a shame. The `--inspect` allows us to attach to the node debugger so that we can do stepwise debugging within node.js of server-side rendered code paths. It's not critical right now since the site is well established and we are unlikely to need to do that kind of debugging, but would be cool if we get it back in at some point :)